Tasks/AzureIoTEdgeV2/taskerror.ts

```diff
@@ -0,0 +1,9 @@
+export class TaskError extends Error {
```

Comment:
Use TaskError to record error summary without details (e.g. file name), since we don't require such information. The prompt message is the full error message that shows in Azure Pipelines log.